app/scripts/script.js

```diff
@@ -0,0 +1,69 @@
+// DateTime Dsplay
+function startTime() {
+    var today = new Date();
+    document.getElementById('datetime').innerHTML =
+     today;
+    var t = setTimeout(startTime, 500);
+  }
+  function checkTime(i) {
+    if (i < 10) {i = "0" + i};  // add zero in front of numbers < 10
+    return i;
+  }
+
+// Function that starts the clock on page load
+  document.getElementById("body").addEventListener("onload", startTime());
+
+// Notepad Functions
+
+// When window loads, execute getNote function
+window.onload = getNote();
+
+// add event handlers for saving and clearing the note
+// Event listener for saving the notepad
+document.getElementById('notepad').addEventListener('keyup', saveNote);
+
+// Event listener for clearing the notepad
+document.getElementById('clearbtn').addEventListener("click", clearNote);
+
+// getnote function
+  function getNote() {
+     // Retrieve the JSON string
+     var storedObj = localStorage.getItem("note");
```

Comment:
`"note"` here should be placed in a constant. That way, you can rename the key any time by simply changing a single variable rather than hunting through your code for instances of the string `"note"`